<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRelReferenceConfigDlgBase</class>
 <widget class="QWidget" name="QgsRelReferenceConfigDlgBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Display expression</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsFieldExpressionWidget" name="mExpressionWidget" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Relation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="mComboRelation"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="mCbxAllowNull">
     <property name="text">
      <string>Allow NULL value</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QCheckBox" name="mCbxShowForm">
     <property name="text">
      <string>Show embedded form</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QCheckBox" name="mCbxMapIdentification">
     <property name="text">
      <string>On map identification (for geometric layers only)</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QCheckBox" name="mCbxReadOnly">
     <property name="text">
      <string>Use a read-only line edit instead of a combobox</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="mCbxOrderByValue">
     <property name="text">
      <string>Order by value</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldExpressionWidget</class>
   <extends>QWidget</extends>
   <header location="global">qgsfieldexpressionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
